src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageReadmeControl.xaml

```diff
@@ -30,11 +30,11 @@
       x:Name="descriptionMarkdownPreview"
       ClipToBounds="True"
       Focusable="False"
-      Visibility="{Binding Path=IsReadmeReady, Mode=OneWay, FallbackValue=Collapsed, Converter={StaticResource BooleanToVisibilityConverter}}" />
+      Visibility="{Binding ElementName=PackageReadmeUserControl, Path=IsReadmeReady, Mode=OneWay, FallbackValue=Collapsed, Converter={StaticResource BooleanToVisibilityConverter}}" />
```

Comment:
We discussed offline:
This will also go away once a all the properties can be combined into a single ViewModel.